hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -97,15 +97,15 @@
       tests: []
     }
     {
-      name: fifo_eviction
+      name: rd_buff_eviction
```

Comment:
This is already committed. Could you rebase this PR, or remove this change?

Reply:
Ok, fixed.